src/core_types.jl

```diff
@@ -11,6 +11,8 @@ Base.@pure StaticArrays.Size(::Type{Rotation{N,T}}) where {N,T} = Size(N,N)
 Base.@pure StaticArrays.Size(::Type{R}) where {R<:Rotation} = Size(supertype(R))
 Base.adjoint(r::Rotation) = inv(r)
 Base.transpose(r::Rotation{N,T}) where {N,T<:Real} = inv(r)
+Base.zero(T::Type{<:Rotation}) = throw(ArgumentError("There is no additive identity for Rotation. Consider using the one($T) method instead."))
+Base.zero(r::Rotation) = throw(ArgumentError("There is no additive identity for Rotation. Consider using the one($(typeof(r))($r)) method instead."))
```

Comment:
See comment at https://github.com/JuliaGeometry/Rotations.jl/issues/157#issuecomment-844294269

Reply:
Here's an updated error message:
```julia
"$T is a subtype of AbstractMatrix, but there is no additive identity for Rotation. To get a non-rotation instance, consider using the one($T) method instead."
```
Is there any problem with this? I'm not native English speaker, so there may be some mistakes. :)